### PR TITLE
[doc][yba][DOC-466] HA certificate chain

### PR DESCRIPTION
**Pull request by ddhodge**
HA certificate chain
DOC-424 - Grafana dashboard

@netlify /preview/yugabyte-platform/administer-yugabyte-platform/high-availability/

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/alerts-monitoring/prometheus-custom/_index.md [hunk 1]
```diff
@@ -25,3 +25,12 @@ You can do this in the following ways:
 - [Scrape directly from universe nodes](prometheus-scrape/)
 
     Your independent Prometheus instance can scrape data from database nodes directly (in the case of VM-based universes) or scrape data from a Kubernetes Prometheus Operator Service Monitor (in the case of K8s-based universes). This data scraping runs in parallel to and independently from the YugabyteDB Anywhere-embedded Prometheus.
+
```
Comment (nbhatia-yb) [inline docs/content/preview/yugabyte-platform/alerts-monitoring/prometheus-custom/_index.md:28]: This URL should be something like

curl -k --request GET   --url https://portal.dev.yugabyte.com/api/v1/grafana_dashboard   --header 'Accept: application/json'  --header 'X-AUTH-YW-API-TOKEN: 123'

**Review by nbhatia-yb — APPROVED** [review]
Looks good Dwight. Just one comment.

**Review by aishwarya24 — APPROVED** [review]
LGTM

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/high-availability.md [hunk 2]
```diff
@@ -144,20 +144,45 @@ For example, if your metrics retention is 14 days on your active instance, and y
...
-    - Add the CA certificates for both the active and standby instances to the YugabyteDB Anywhere trust store of the active instance. This allows a standby to connect to the active instance if the standby is promoted to active status.
+    **Auto-generated CA certificates**
+
+    If YBA was set up to use automatically generated self-signed certificates (the default), the CA certificate is in the following location on the active and standby instances:
+
+    | Installation | Certificate Location |
+    | :--- | :--- |
+    | YBA Installer | `/opt/yugabyte/data/yba-installer/certs/ca_cert.pem` <br/>If you configured a [custom install root](../../install-yugabyte-platform/install-software/installer/#configuration-options), replace `/opt/yugabyte` with the path you configured. |
+    | Replicated | `/var/lib/replicated/secrets/ca.crt` |
+    | Kubernetes | Locate the CA .pem file by running the following command:<br/>`kubectl get secret -n <namespace> <helm-release-name>-yugaware-tls-pem -o jsonpath="{.data['ca\.pem']}" \| base64 -d`<br/>Replace `<namespace>` and `<helm_release_name>` with appropriate values. |
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/high-availability.md:159]:
```suggestion
    | Kubernetes | Locate the `CA.pem` file by running the following command:<br/>`kubectl get secret -n <namespace> <helm-release-name>-yugaware-tls-pem -o jsonpath="{.data['ca\.pem']}" \| base64 -d`<br/>Replace `<namespace>` and `<helm_release_name>` with appropriate values. |
```

File: docs/content/stable/yugabyte-platform/administer-yugabyte-platform/high-availability.md [hunk 3]
```diff
@@ -141,20 +141,45 @@ For example, if your metrics retention is 14 days on your active instance, and y
...
-    - Add the CA certificates for both the active and standby instances to the YugabyteDB Anywhere trust store of the active instance. This allows a standby to connect to the active instance if the standby is promoted to active status.
+    **Auto-generated CA certificates**
+
+    If YBA was set up to use automatically generated self-signed certificates (the default), the CA certificate is in the following location on the active and standby instances:
+
+    | Installation | Certificate Location |
+    | :--- | :--- |
+    | YBA Installer | `/opt/yugabyte/data/yba-installer/certs/ca_cert.pem` <br/>If you configured a [custom install root](../../install-yugabyte-platform/install-software/installer/#configuration-options), replace `/opt/yugabyte` with the path you configured. |
+    | Replicated | `/var/lib/replicated/secrets/ca.crt` |
+    | Kubernetes | Locate the CA .pem file by running the following command:<br/>`kubectl get secret -n <namespace> <helm-release-name>-yugaware-tls-pem -o jsonpath="{.data['ca\.pem']}" \| base64 -d`<br/>Replace `<namespace>` and `<helm_release_name>` with appropriate values. |
```
Comment (aishwarya24) [inline docs/content/stable/yugabyte-platform/administer-yugabyte-platform/high-availability.md:156]:
```suggestion
    | Kubernetes | Locate the `CA.pem` file by running the following command:<br/>`kubectl get secret -n <namespace> <helm-release-name>-yugaware-tls-pem -o jsonpath="{.data['ca\.pem']}" \| base64 -d`<br/>Replace `<namespace>` and `<helm_release_name>` with appropriate values. |
```

**Review by aishwarya24 — APPROVED** [review]

**Pull request merged**